db/controllers.js

```diff
@@ -0,0 +1,11 @@
+const controllers={};
+const models= require('./models');
+
+controllers.getTopics = (req,res) => {
```

Comment:
So rather than exporting it on your own controllers object, you could cut out some work and export everything on the exports key aka `exports.getTopics = ...` and when you require it in just require the exported object as controllers just like you have on line 2 of the app.js file